.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "useTabs": false,
+  "tabWidth": 2,
+  "singleQuote": true
+}
```

Comment:
This commits to a specific tool to maintain this formatting as well. Is this the best tool?